OnlineTradePlatform/OnlineTradePlatform.Application/IServices/IAdService.cs

```diff
@@ -15,6 +17,6 @@ public interface IAdService
 
         Task<PagedList<AdDto>> GetAdPageAsync(PageParameters pageParameters, Expression<Func<Ad, bool>> predicate, CancellationToken cancellationToken);
     
-        Task DeleteAdAsync(int id, CancellationToken cancellationToken);
+        Task DeleteAdAsync(AdDto dto, CancellationToken cancellationToken);
```

Comment:
leave int id as a parameter and in implementation get entity from db and delete it